Util/BuildTools/Setup.sh

```diff
@@ -217,7 +217,7 @@ unset BOOST_BASENAME
 # -- Get rpclib and compile it with libc++ and libstdc++ -----------------------
 # ==============================================================================
 
-RPCLIB_PATCH=v2.2.1_c5
+RPCLIB_PATCH=carla-callbacks
```

Comment:
Here we should enter v2.2.1_c6 after a corresponding tag at the head of the branch 
https://github.com/carla-simulator/rpclib/tree/carla-callbacks
is created.